dll/cpl/inetcpl/lang/en-US.rc

```diff
@@ -66,7 +66,7 @@ CAPTION "Delete browsing history"
 BEGIN
 
     AUTOCHECKBOX   "&Temporary internet files\nCached copies of web pages, images and certificates.",
-                    IDC_DELETE_TEMP_FILES, 10, 8, 230, 40, BS_TOP | BS_MULTILINE
+                    IDC_DELETE_TEMP_FILES, 10, 8, 230, 40, BS_TOP | BS_MULTILINE | WS_GROUP | WS_TABSTOP
```

Comment:
So the explicit WS_TABSTOP is necessary now? OK why not, still strange, but at least there are many less changes for fixing the navigation, so I'm fine with it :+1:

Reply:
Group does not imply TabStop AFAIK

Reply:
Yeah, when specifying WS_GROUP It removes the default WS_TABSTOP